data/json/items/classes/chainmail.json

```diff
@@ -0,0 +1,97 @@
+[
+  {
+    "abstract": "base_chainmail_vest",
```

Comment:
These might be more readable to stick in `torso_armor.json`? It's only impacting things in that file, so then I don't need to look between two files to find the stats on these items.